README.md

```diff
@@ -58,24 +58,24 @@ This extension uses three main things in its build process:
 - Webpack: Bundles entry points for all Web Extension pages (background, popup, options,
   content scripts) into single files and manages most of the build process.
 
-1. Clone the repository.
-2. Install Node.js (>=18.12.1) and Yarn (>= v1.10.0).
+1.  Clone the repository.
```

Comment:
Interesting. I guess there is an autoformatter that ran as a pre-commit hook and changed this. I didn't add the double space between each number and its text.